### DISCUSSION TITLE
Best way to mark a test / test suite as "browser only"

### DISCUSSION THREAD

**Discussion by epreston** [vitest-dev/vitest#3327] [Q&A] [answered]
Hello there,

What's the best way to mark a test or suite of tests grouped by describe as "run during browser testing only".

I'm open to decorators like "describe.todo", clever configurations, or segmenting tests into specific files.  Looking for some best practice.

Thank you.

**Comment by sheremet-va** [comment, marked as answer]
In my opinion, the best solution would be to create a separate workspace in `vitest.workspace.ts`:

```ts
export default [
  { extends: './vitest.config.ts' }, // to run other tests 
  { extends: './vitest.config.ts', test: { include: ['**/browser.test.ts'], browser: { enabled: true } } }, // to run browser tests 
]
```
